rasa/importers/autoconfig.py

```diff
@@ -0,0 +1,241 @@
+    except ValueError:
+        content = ""
+
+    language_to_overwrite = None
+    if not content:
+        content = _create_and_read_config_file(config_file_path)
+        # if the config file was empty or not present, the default language will be
+        # overwritten with the language in the current config
+        language_to_overwrite = config.get("language")
+
+    missing_keys = _get_missing_config_keys(content)
+    _add_missing_config_keys_to_file(missing_keys, config_file_path)
+
+    autoconfigured = _get_unspecified_autoconfigurable_keys(content)
+    autoconfig_lines = _get_commented_out_autoconfig_lines(config, autoconfigured)
+
+    try:
+        with open(config_file_path, "r+", encoding=io_utils.DEFAULT_ENCODING) as f:
+            lines = f.readlines()
+            updated_lines = _get_lines_including_autoconfig(
+                lines, autoconfig_lines, language_to_overwrite
+            )
+            f.seek(0)
+            for line in updated_lines:
+                f.write(line)
+    except FileNotFoundError:
+        raise ValueError(f"File '{config_file_path}' does not exist.")
+
+    if autoconfigured:
+        autoconfigured_keys = common_utils.transform_collection_to_sentence(
+            autoconfigured
+        )
+        cli_utils.print_info(
+            f"The configuration for {autoconfigured_keys} was chosen automatically. It "
+            f"was written into the config file at `{config_file_path}`."
+        )
+
+
+def _create_and_read_config_file(config_file_path: Text) -> Dict[Text, Any]:
+    import pkg_resources
+
+    cli_utils.print_warning(
+        f"Configuration file {config_file_path} does not exist or is empty or invalid. "
+        f"Creating a new one now and filling it with the current configuration."
+    )
+
+    empty_config_file = pkg_resources.resource_filename(
+        "rasa.cli.initial_project", "config.yml"
+    )
+    shutil.copy(empty_config_file, config_file_path)
+
+    return io_utils.read_config_file(config_file_path)
+
+
+def _add_missing_config_keys_to_file(
+    missing_keys: List, config_file_path: Text
+) -> None:
+    if missing_keys:
+        with open(config_file_path, "a", encoding=io_utils.DEFAULT_ENCODING) as f:
+            for key in missing_keys:
+                f.write(f"{key}:\n")
+
+
+def _get_lines_including_autoconfig(
+    lines: List[Text],
+    autoconfig_lines: Dict[Text, List[Text]],
+    language_to_overwrite: Text = None,
+) -> List[Text]:
+    autoconfigured = autoconfig_lines.keys()
+
+    lines_with_autoconfig = []
+    remove_comments_until_next_uncommented_line = False
+    for line in lines:
+        insert_section = None
+
+        # overwrite language if necessary
+        if language_to_overwrite and re.match("language:", line):
+            line = f"language: {language_to_overwrite}\n"
+
+        # remove old auto configuration
+        if remove_comments_until_next_uncommented_line:
+            if re.match("#", line):
```

Comment:
?
```suggestion
            if re.startswith("#", line):
```

Reply:
`line.startswith("#")` and we can completely get rid of importing `re`